etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -1,76 +1,83 @@
+# Modules ##################################################
 #
 module purge
 module load modenv/ml
-# similar to foss/2018a, but also includes SpectrumMPI which basically is just a OpenMPI-fork of IBM
-module load gsolf/2018a
-module load GCC/6.4.0-2.28
-module load CMake/3.10.2-GCCcore-6.4.0
-# CUDA is no module in the current enviroment!
-#module load CUDA/9.2.88  # gcc <= 7, intel 15-17
-# OpenMPI is already loaded
-#module load OpenMPI/2.1.2-GCC-6.4.0-2.28
+
+# load CUDA/9.2.88-GCC-7.3.0-2.30, also loads GCC/7.3.0-2.30, zlib, OpenMPI and others
+module load fosscuda/2018b
+module load CMake/3.11.4-GCCcore-7.3.0
 module load git/2.18.0-GCCcore-6.4.0
-module load zlib/1.2.11-GCCcore-6.4.0
+module load libpng/1.6.34-GCCcore-7.3.0
 
-# Self-Build Software #########################################################
+printf "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n"
+printf "@ Note: You need to compile picongpu on a node. @\n"
+printf "@       Similar for building the libraries.     @\n"      
+printf "@       Get a node with the getNode command.    @\n"
+printf "@       Then source %s again.@\n" "$(basename $PIC_PROFILE)"
+printf "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@\n"
+
+# Self-Build Software #####################################
 #
 # needs to be compiled by the user
-export PIC_LIBS="$HOME/lib"
-export BOOST_ROOT=$PIC_LIBS/boost-1.69.0-Power9
-export PNG_ROOT=$PIC_LIBS/libpng-1.6.34-Power9
+# Check the install script at
+# (https://gist.github.com/steindev/cc02eae81f465833afa27fc8880f3473#file-picongpu_0-4-3_taurus-tud-sh)
+#
+export PIC_LIBS="/scratch/p_electron/lib/power9"
+export BOOST_ROOT=$PIC_LIBS/boost-1.68.0-Power9
 export PNGwriter_DIR=$PIC_LIBS/pngwriter-0.7.0-Power9
 export ADIOS_ROOT=$PIC_LIBS/adios-1.13.1-Power9
+export HDF5_ROOT=$PIC_LIBS/hdf5-1.8.20-Power9
 export Splash_DIR=$PIC_LIBS/splash-Power9
-export CMAKE_PREFIX_PATH=$Splash_DIR:$CMAKE_PREFIX_PATH
-export HDF5_ROOT=$PIC_LIBS/hdf5-Power9
-export BLOSC_ROOT=$PIC_LIBS/blosc-1.12.1-Power9
+export BLOSC_ROOT=$PIC_LIBS/blosc-1.16.2-Power9
 
 export LD_LIBRARY_PATH=$BOOST_ROOT/lib:$LD_LIBRARY_PATH
-export LIBRARY_PATH=$BOOST_ROOT/lib:$LIBRARY_PATH
-export LD_LIBRARY_PATH=$PNG_ROOT/lib:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$PNGwriter_DIR/lib:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$ADIOS_ROOT/lib:$LD_LIBRARY_PATH
-export LD_LIBRARY_PATH=$Splash_DIR/lib:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$HDF5_ROOT/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$Splash_DIR/lib:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$BLOSC_ROOT/lib:$LD_LIBRARY_PATH
 
-export PATH=$PNG_ROOT/bin:$PATH
 export PATH=$ADIOS_ROOT/bin:$PATH
 
-export CMAKE_PREFIX_PATH=$PNG_ROOT:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=$HDF5_ROOT:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=$Splash_DIR:$CMAKE_PREFIX_PATH
 
-export PICSRC=$HOME/src/picongpu
-export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
+export PICSRC="$HOME/src/picongpu"
```

Comment:
The quotes `"` are not required.

Reply:
please remove the unnecessary `""` in all locations, also below :)